ajax/libs/most/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "most",
+  "filename": "most.min.js",
+  "version": "0.19.7",
+  "description": "Monadic streams",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cujojs/most"
+  },
+  "keywords": [
+    "reactive",
+    "reactive programming",
+    "reactive streams",
+    "streams",
+    "event stream",
+    "promises",
+    "promises-aplus",
+    "fantasy-land",
+    "monad",
+    "monadic",
+    "functional",
+    "async",
+    "cujojs",
+    "cujo"
+  ],
+  "author": "Brian Cavalier <brian@hovercraftstudios.com>",
+  "license": "MIT",
+  "dependencies": {
+    "@most/multicast": "^1.1.1",
+    "@most/prelude": "^1.2.0",
+    "symbol-observable": "^0.2.4"
+  },
+  "npmName": "most",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
I didn't see `dist` folder in npm package `most` v0.19.7, @kennynaoh @dannyxx001 are you sure about this?